packages/gardens/src/components/TopNav.tsx

```diff
@@ -181,15 +182,15 @@ function TopNav() {
                 </Box>
               </Link>
               <Link
-                as={ReachLink}
-                to="https://medium.com/@shrubfinance"
+                href="https://medium.com/@shrubfinance"
+                isExternal
                 variant="ghost"
                 cursor="pointer"
                 px={2}
                 py={{ base: "3", md: "1", lg: "1" }}
                 rounded={"lg"}
                 _hover={{ textDecoration: "none", bgGradient: gradient }}
-                onClick={onMenuClose}
+                onClick={handleGA}
```

Comment:
we may still need the `onMenuClose` may be create a method like 

onClick={handleClick} and move these 2 methods there